recipes/openal/all/patches/1.21.0-c++14-does-not-have-std-aligned_alloc.patch

```diff
@@ -0,0 +1,16 @@
+gcc-11 triggers an error. std::aligned_alloc needs c++17.
+This patch ports openal 1.21.1 behavior back to 1.21.0.
+--- common/almalloc.cpp
++++ common/almalloc.cpp
+@@ -21,4 +21,4 @@
+-#if defined(HAVE_STD_ALIGNED_ALLOC)
```

Comment:
it's c++17, but why HAVE_STD_ALIGNED_ALLOC is defined? shouldn't it detect the availability of the function?
I guess it should be fixed in another place

Reply:
openal is compiled as c++14.
I guess it was fine for gcc-10 but gcc-11 is causing problems.

Reply:
https://github.com/kcat/openal-soft/blob/62e6ce6d7322bc83ba3150316d9c921f5fadbb99/CMakeLists.txt#L156-L159

Reply:
this seems to be an upstream fix?
https://github.com/kcat/openal-soft/commit/f576a353638236775ecb4f4e8ba93cb1716fbe2d

Reply:
Looks like it. I didn't look at the commit history and looked only at the diff around the `alogned_alloc`